try:
  print(x)
except:
  print("Exception")
